### PR TITLE
Add _make_streaming_request, formerly _make_streaming_config.

### PR DESCRIPTION
**Pull request by daspecster**
Since this actually returned `StreamingRecognizeRequest`, I renamed it to `_make_streaming_request`.

### CODE REVIEW

File: speech/unit_tests/test__gax.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            SpeechContext)
+        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+            StreamingRecognizeRequest)
+
+        sample = Sample(source_uri=self.AUDIO_SOURCE_URI,
+                        encoding=speech.Encoding.FLAC,
+                        sample_rate=self.SAMPLE_RATE)
+        streaming_request = self._callFUT(sample, 'US-en', 2, True,
+                                          SpeechContext(), True, True)
+        self.assertIsInstance(streaming_request, StreamingRecognizeRequest)
```
Comment (dhermes) [inline speech/unit_tests/test__gax.py]: You don't want to make any assertions about the properties?

File: speech/unit_tests/test__gax.py [hunk 2]
```diff
@@ -39,11 +38,38 @@ def test_ctor(self):
...
+        speech_context = SpeechContext(phrases=self.HINTS)
+        single_utterance = True
+        interim_results = False
+
+        streaming_request = self._callFUT(sample, language_code,
+                                          max_alternatives, profanity_filter,
+                                          speech_context, single_utterance,
+                                          interim_results)
         self.assertIsInstance(streaming_request, StreamingRecognizeRequest)
+        self.assertIsInstance(streaming_request.audio_content, bytes)
```
Comment (dhermes) [inline speech/unit_tests/test__gax.py]: It's overkill to check simple types like this, why not just check the value?

Comment (daspecster) [inline speech/unit_tests/test__gax.py]: So at this point, `streaming_request.audio_content` should be empty `b''`. The first `StreamingRecognizeRequest` **must** not have `audio_content` set and only `streaming_config` set.
The second `StreamingRecognizeRequest` can only have `audio_content` set and not `streaming_config.

I added a comment about this in the test for now.

**Comment by daspecster** [comment]
Also just squashed.

**Pull request merged**